eval/scripts/run_eval.py

```
@@ -0,0 +1,136 @@
+
+
+def normalize(text: str) -> str:
+    text = text.lower()
+    text = _PUNCT.sub(" ", text)
+    text = _APOS.sub(" ", text)
+    return _WS.sub(" ", text).strip()
+
+
+def transcribe(cli: str, wav: Path, model: str) -> tuple[str, float, float]:
+    start = time.monotonic()
+    proc = subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)